palette/src/rgb/rgb.rs

```diff
@@ -118,6 +118,22 @@ impl<S: RgbStandard, T: Component> Rgb<S, T> {
     }
 }
 
+impl<S: RgbStandard> Rgb<S, u8> {
+    /// Convert from a packed `u32` with specifiable component order. Defaults
+    /// to ARGB ordering (0x00RRGGBB).
+    ///
+    /// See [PackedRgb](struct.PackedRgb.html) for more details.
+    pub fn from_hex<C: RgbChannels>(color: u32) -> Self {
+        PackedRgb::<C>::from(color).into()
+    }
+
+    /// Convert to a [PackedRgb](struct.PackedRgb.html) with color packed into
+    /// a `u32`. Defaults to ARGB ordering (0x00RRGGBB).
+    pub fn into_hex<C: RgbChannels>(self) -> PackedRgb<C> {
+        PackedRgb::from(self)
+    }
```

Comment:
These are asymmetric now, when we take `u32` as input, but give `PackedRgb` as output. I think it's redundant to give out `PackedRgb` in this case, since it can easily be replaced by `PackedRgb::from(my_rgb)`. Thinking a bit more about it, it may be better to call them `{from|into}_u32` to be really explicit about it, and also perhaps implement `From<u32>` with the ARGB default.

Reply:
I agree they shouldn't be asymmetric and I originally wrote a blurb in my last comment as a concern but got rid of it. 

I went back and forth on u32 or the struct return. I think returning the struct makes the user have to reason about the channel ordering more which can be a good thing but `into_u32` should return a u32 if we're changing the method name.